pyogrio/tests/test_geopandas_io.py

```diff
@@ -704,3 +704,26 @@ def test_custom_crs_io(tmpdir, naturalearth_lowres_all_ext):
     assert crs["lat_2"] == 51.5
     assert crs["lon_0"] == 4.3
     assert df.crs.equals(expected.crs)
+
+
+@pytest.mark.parametrize("ext", ALL_EXTS)
```

Comment:
Would it be sufficient to test with a single driver in this case? 
The code affected is purely on the python (cython) side, determining when to indicate to GDAL to set a null or not, which shouldn't have driver specific behaviour? 

(I know it is not always easy in advance to predict when something actually runs into a driver specific issue, so we might miss things this way, but I also want to avoid the test suite becoming unnecessarily bloated; it's nice that the tests run fast)

Reply:
I added the different file types to check that for all formats the roundtrip of reading + writing the null/nan/None values behave the same... but possibly we can trust gdal to show consistent behaviour for this, so I'll remove it.